src/query/functions/tests/it/scalars/testdata/function_list.txt

```diff
@@ -2580,228 +2580,228 @@ Functions overloads:
-162 minus(UInt32, Float32) :: Float64
-163 minus(UInt32 NULL, Float32 NULL) :: Float64 NULL
-164 minus(UInt32, Float64) :: Float64
-165 minus(UInt32 NULL, Float64 NULL) :: Float64 NULL
-166 minus(UInt64, Float32) :: Float64
-167 minus(UInt64 NULL, Float32 NULL) :: Float64 NULL
-168 minus(UInt64, Float64) :: Float64
-169 minus(UInt64 NULL, Float64 NULL) :: Float64 NULL
-170 minus(Int8, Float32) :: Float64
-171 minus(Int8 NULL, Float32 NULL) :: Float64 NULL
-172 minus(Int8, Float64) :: Float64
-173 minus(Int8 NULL, Float64 NULL) :: Float64 NULL
-174 minus(Int16, Float32) :: Float64
-175 minus(Int16 NULL, Float32 NULL) :: Float64 NULL
-176 minus(Int16, Float64) :: Float64
-177 minus(Int16 NULL, Float64 NULL) :: Float64 NULL
-178 minus(Int32, Float32) :: Float64
-179 minus(Int32 NULL, Float32 NULL) :: Float64 NULL
-180 minus(Int32, Float64) :: Float64
-181 minus(Int32 NULL, Float64 NULL) :: Float64 NULL
-182 minus(Int64, Float32) :: Float64
-183 minus(Int64 NULL, Float32 NULL) :: Float64 NULL
-184 minus(Int64, Float64) :: Float64
-185 minus(Int64 NULL, Float64 NULL) :: Float64 NULL
-186 minus(Float32, UInt8) :: Float64
-187 minus(Float32 NULL, UInt8 NULL) :: Float64 NULL
-188 minus(Float32, UInt16) :: Float64
-189 minus(Float32 NULL, UInt16 NULL) :: Float64 NULL
-190 minus(Float32, UInt32) :: Float64
-191 minus(Float32 NULL, UInt32 NULL) :: Float64 NULL
-192 minus(Float32, UInt64) :: Float64
-193 minus(Float32 NULL, UInt64 NULL) :: Float64 NULL
-194 minus(Float32, Int8) :: Float64
-195 minus(Float32 NULL, Int8 NULL) :: Float64 NULL
-196 minus(Float32, Int16) :: Float64
-197 minus(Float32 NULL, Int16 NULL) :: Float64 NULL
-198 minus(Float32, Int32) :: Float64
-199 minus(Float32 NULL, Int32 NULL) :: Float64 NULL
-200 minus(Float32, Int64) :: Float64
-201 minus(Float32 NULL, Int64 NULL) :: Float64 NULL
-202 minus(Float64, UInt8) :: Float64
-203 minus(Float64 NULL, UInt8 NULL) :: Float64 NULL
-204 minus(Float64, UInt16) :: Float64
-205 minus(Float64 NULL, UInt16 NULL) :: Float64 NULL
-206 minus(Float64, UInt32) :: Float64
-207 minus(Float64 NULL, UInt32 NULL) :: Float64 NULL
-208 minus(Float64, UInt64) :: Float64
-209 minus(Float64 NULL, UInt64 NULL) :: Float64 NULL
-210 minus(Float64, Int8) :: Float64
-211 minus(Float64 NULL, Int8 NULL) :: Float64 NULL
-212 minus(Float64, Int16) :: Float64
-213 minus(Float64 NULL, Int16 NULL) :: Float64 NULL
-214 minus(Float64, Int32) :: Float64
-215 minus(Float64 NULL, Int32 NULL) :: Float64 NULL
-216 minus(Float64, Int64) :: Float64
-217 minus(Float64 NULL, Int64 NULL) :: Float64 NULL
-218 minus(Float32, Float32) :: Float64
-219 minus(Float32 NULL, Float32 NULL) :: Float64 NULL
-220 minus(Float32, Float64) :: Float64
-221 minus(Float32 NULL, Float64 NULL) :: Float64 NULL
-222 minus(Float64, Float32) :: Float64
-223 minus(Float64 NULL, Float32 NULL) :: Float64 NULL
-224 minus(Float64, Float64) :: Float64
-225 minus(Float64 NULL, Float64 NULL) :: Float64 NULL
+4 minus FACTORY
```

Comment:
Something is changed here.

Reply:
The function order should be consistent if we only refactor the crates by spiliting.

Reply:
Get